lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2375,6 +2375,9 @@ def bar3d(self, x, y, z, dx, dy, dz, color=None,
         shade : bool, optional (default = True)
             When true, this shades the dark sides of the bars (relative
             to the plot's source of light).
+            
```

Comment:
This is the line that causes the flake hickup. A blank like cannot contain any whitespaces.